mono/monodevelop
# Added Git Tests

Tests Added:

* Commit successful
* Stash should notify if there is no changes
* Stash should not work if there is no HEAD commit
* Stash successful
* Unstash successful
* Create new Branch
* Switch Branch
* Edit Branch
* Select Remote
* Add Remote
* Edit Remote (disabled for now)
* Fetch Remote Branches
* Track Remote Branch in Local
